### PR TITLE
Update tags on marketplace

### PR DESCRIPTION
**Pull request by jasmingoh**

### CODE REVIEW

**Review by Tanya0609 — DISMISSED** [review]

File: src/extension/package.json [hunk 1]
```diff
@@ -17,12 +17,21 @@
...
+    "Templates",
     "Boilerplate",
     "React",
+    "Angular",
+    "Vue.js",
+    "Node",
     "Express",
-    "Scaffolding"
+    "Flask",
+    "Azure services",
```
Comment (jsondoo) [inline src/extension/package.json]: should we do Azure Services to be consistent?

File: src/extension/package.json [hunk 2]
```diff
@@ -17,12 +17,21 @@
   },
   "icon": "resources/WebTSLogoSquare.png",
   "keywords": [
-    "Garage",
+    "Open source",
+    "Full stack",
```
Comment (jsondoo) [inline src/extension/package.json]: same here 
Open Source
Full Stack

**Review by streetcoder123 — APPROVED** [review]

**Review by jsondoo — APPROVED** [review]

**Review by annielovesjs — APPROVED** [review]

**Review by annielovesjs — COMMENTED** [review]
do you want to change vue.js to vue to make it more consistent?

**Comment by jasmingoh** [comment]
We are sticking with Vue.js since that's the official name

**Pull request merged**